sycl/include/sycl/detail/pi.h

```diff
@@ -2463,6 +2463,38 @@ __SYCL_EXPORT pi_result piextCommandBufferMemBufferWriteRect(
     const pi_ext_sync_point *sync_point_wait_list,
     pi_ext_sync_point *sync_point);
 
+/// API to append a USM Prefetch command to the command-buffer.
+/// \param command_buffer The command-buffer to append onto.
+/// \param ptr points to the memory to migrate.
+/// \param size is the number of bytes to migrate.
+/// \param flags is a bitfield used to specify memory migration options.
+/// \param num_sync_points_in_wait_list The number of sync points in the
+/// provided wait list.
+/// \param sync_point_wait_list A list of sync points that this command must
+/// wait on.
+/// \param sync_point The sync_point associated with this memory operation.
+__SYCL_EXPORT pi_result piextCommandBufferPrefetchUSM(
```

Comment:
We should bump `_PI_H_VERSION_MINOR ` at the top of this file with a comment that we're adding these two new entry-points